en/docs/getting-started/basic-architecture.md

```diff
@@ -0,0 +1,78 @@
+
+
+## Data Plane
+
+Data plane is where the created API is exposed to the public consumers and acts as the proxy for API calls. This also provides additional capabilities such as enforcing security, rate limiting etc. (policy enforcement point).
+
+### Components
+
+#### API Gateway
+
+![API Gateway]({{base_path}}/assets/img/learn/gateway-overview.png)
+
+WSO2 API Gateway acts as the entry point for an API request made to an API managed by WSO2 API Manager.
+
+JWT token validation is done by the API Gateway. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map. This in-memory map includes API-related information, application-related information, subscription-related information, etc. which is updated each time an artifact (API/application) is updated.
+  
+ Once the token is validated, the API Gateway acts upon the API request before sending it to the backend. It first processes the message to a pre-configured format (e.g. JSON/XML/CSV/etc.).  It then applies security policies, rate-limiting policies,  collects statistics, etc., via its handlers.  The mediators then act upon the API payload based on the mediation logic developed. The message is then formatted to a pre-configured format (e.g.JSON/XML/CSV/etc.) and sent to the backend. The WSO2 API Gateway supports transports such as HTTP, HTTPS, etc. It is also able to scale on-demand in cloud environments and is easily pluggable in non-cloud environments. 
+ 
+## Control Plane
+ 
+ Control plane is where the API security validation, API key generation and rate limiting decisions are taken.
+ 
+### Components
+ 
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
+
+ The Key Manager is the identity provider for the WSO2 API Manager and acts as the Secure Token Service(STS). 
+ 
+ In WSO2 API Manager, tokens are generated for an application. The Key manager provides a token API to generate access tokens. These tokens can be used by clients to invoke APIs exposed by WSO2 API Manager. Furthermore, 
+ The Key Manager exposes a revoke API to revoke any access token. The Key Manager supports OAuth2.0, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms. 
+ 
+ WSO2 API Manager only supports creating applications that support JWT formatted OAuth2.0 type tokens.
+ A client can generate a JWT formatted OAuth 2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, an API Key can be generated through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token.
+ 
+ When a client invokes an API with a JWT formatted OAuth2.0 access token or an API Key,  the Gateway validates the token by validating its signature and subscription.
```

Comment:
Please remove all references of "JWT formatted access tokens". The format of the token doesn't have to be highlighted in this manner because we aren't doing anything special.